upstream/roles/install_iib/templates/Dockerfile-workers.j2

```diff
@@ -75,6 +75,6 @@ RUN sed -i -e 's|^#mount_program|mount_program|g' /etc/containers/storage.conf
 COPY docker/libpod.conf /usr/share/containers/libpod.conf
 
 COPY . .
-RUN pip3 install -r requirements.txt --no-deps --require-hashes
-RUN pip3 install . --no-deps
+RUN pip3 install --no-cache-dir -r requirements.txt --no-deps --require-hashes
+RUN pip3 install --no-cache-dir . --no-deps
```

Comment:
You might want to merge two RUn commands to the single one 
```
RUN pip3 install --no-cache-dir -r requirements.txt --no-deps --require-hashes \
    && pip3 install --no-cache-dir . --no-deps
```

Reply:
I think this way is fine, as it is installing dependencies, using "requirements.txt" in the first line then installing the actual package later.

Reply:
The description of PR mention best practices.
And this is one of best practices.
* https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers

And you you decided to touch lines it would be so such simple change as well.

Reply:
@lslebodn  All operation is happening in a single "RUN" using "&&", so it will not create a new layer.

Reply:
I can still see two `RUN` instructions in the diff. Maybe you forgot to push changes :-)

Reply:
Sorry @lslebodn my bad, corrected and pushed